core/lib/Thelia/Config/Resources/routing/admin.xml

```diff
@@ -417,6 +417,10 @@
         <default key="_controller">Thelia\Controller\Admin\ProductController::deleteProductSaleElementAction</default>
     </route>
 
+    <route id="admin.product.combination.delete-all" path="/admin/product/combination/delete-all">
```

Comment:
This route is vulnerable to CSRF attack, protect it with a token. Use all the form process or the url_token function in your template and verify it in your controller